app/assets/javascripts/webchat/library.js

```diff
@@ -27,14 +23,19 @@
       if (!availabilityUrl || !openUrl) {
         throw Error.new('urls for webchat not defined')
       }
-      $openButton.on('click', handleOpenChat)
+
+      if (redirectUrl === true) {
+        var openButton = document.getElementsByClassName('.js-webchat-open-button')[0]
+        openButton.addEventListener('click', handleOpenChat)
```

Comment:
Suggestion (slightly cleaner and a little more robust - jQuery silently fails if you attach a listener to a non-existent element, JS doesn't)

```
var openButton = document.querySelector('.js-webchat-open-button')
if (openButton) {
  openButton.addEventListener('click', handleOpenChat)
}
```